spec/dummy/db/schema.rb

```diff
@@ -76,6 +76,20 @@
     t.text     "information"
   end
 
+  create_table "departments", force: true do |t|
+    t.string   "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "departments_cities", force: true do |t|
+    t.integer "department_id"
+    t.integer "city_id"
+  end
+
+  add_index "departments_cities", ["city_id"], name: "index_departments_cities_on_city_id"
+  add_index "departments_cities", ["department_id"], name: "index_departments_cities_on_department_id"
```

Comment:
Line is too long. [102/100]

Reply:
I asked rubocop to ignore this file

Reply:
I think you added the wrong path for rubocop to ignore, see the other comment!